pennylane/devices/default_clifford.py

```diff
@@ -0,0 +1,707 @@
+            x_{\left(  2n\right)  1}  & \cdots & x_{\left(  2n\right)  n} & &
+            z_{\left(  2n\right)  1}  & \cdots & z_{\left(  2n\right)  n} & & r_{2n}
+            \end{bmatrix}
+
+        Rows :math:`1` to :math:`n` of the tableau represent the destabilizer generators
+        :math:`R_{1},\ldots,R_{n}`, and rows :math:`n+1` to :math:`2n` represent the stabilizer
+        generators :math:`R_{n+1},\ldots,R_{2n}`. If :math:`R_{i}=\pm P_{1}\ldots P_{n}`,
+        then bits :math:`x_{ij},z_{ij}` determine the j\ :math:`^{th}` Pauli matrix
+        :math:`P_{j}:\ 00` means `I`, :math:`01` means `X`, :math:`11` means `Y`,
+        and :math:`10` means `Z`. Finally, :math:`r_{i}` is :math`1` if :math:`R_{i}`
+        has negative phase and :math:`0` if :math:`r_{i}` has positive phase.
+
+
+    .. details::
+        :title: Tracking
+
+        ``DefaultClifford`` tracks:
+
+        * ``executions``: the number of unique circuits that would be required on quantum hardware
+        * ``shots``: the number of shots
+        * ``resources``: the :class:`~.resource.Resources` for the executed circuit.
+        * ``simulations``: the number of simulations performed. One simulation can cover multiple QPU executions, such as for non-commuting measurements and batched parameters.
+        * ``batches``: The number of times :meth:`~.execute` is called.
+        * ``results``: The results of each call of :meth:`~.execute`
+        * ``derivative_batches``: How many times :meth:`~.compute_derivatives` is called.
+        * ``execute_and_derivative_batches``: How many times :meth:`~.execute_and_compute_derivatives` is called
+        * ``derivatives``: How many circuits are submitted to :meth:`~.compute_derivatives` or :meth:`~.execute_and_compute_derivatives`.
+
+    .. details::
+        :title: Accelerate calculations with multiprocessing
+
+        Suppose one has a processor with 5 cores or more, these scripts can be executed in
+        parallel as follows
+
+        >>> dev = DefaultClifford(max_workers=5)
+        >>> program, execution_config = dev.preprocess()
+        >>> new_batch, post_processing_fn = program(qscripts)
+        >>> results = dev.execute(new_batch, execution_config=execution_config)
+        >>> post_processing_fn(results)
+
+        If you monitor your CPU usage, you should see 5 new Python processes pop up to
+        crunch through those ``QuantumScript``'s. Beware not oversubscribing your machine.
+        This may happen if a single device already uses many cores, if NumPy uses a multi-
+        threaded BLAS library like MKL or OpenBLAS for example. The number of threads per
+        process times the number of processes should not exceed the number of cores on your
+        machine. You can control the number of threads per process with the environment
+        variables:
+
+        * OMP_NUM_THREADS
+        * MKL_NUM_THREADS
+        * OPENBLAS_NUM_THREADS
```

Comment:
```suggestion
        * ``OMP_NUM_THREADS``
        * ``MKL_NUM_THREADS``
        * ``OPENBLAS_NUM_THREADS``
```

Reply:
pls make the change in default.qubit as well 👍